src/main/webapp/js/instructor.js

```diff
@@ -14,6 +14,11 @@ $(document).ready(function() {
     
     // bind the show picture onhover events
     bindStudentPhotoHoverLink('.profile-pic-icon-hover');
+    
+    /* Formatting searchKey for highlight searchKey plugin ( ['string1','string2',...] )*/
+    var searchKey = $('#searchBox').val();
+    var splittedSearchKey = searchKey.split(" ");
```

Comment:
http://dictionary.cambridge.org/us/dictionary/english/split

The use of the word ```splitted``` is archaic, the past tense of ```split``` is ```split```